### DISCUSSION TITLE
Clinvar Update: Clinvar ID blank

### DISCUSSION THREAD

**Discussion by jwehr618** [KarchinLab/open-cravat#285] [Usability & Feature Requests]
With the new Clinvar update to the annotator, the Clinvar ID column is available but it is blank in the output. These are for variants that have entries and have info in other columns just not the ID. This is the ID used to look up the variants on the website. 

Thanks.

**Comment by skanderson** [comment]
Apologies, we had an issue with the Clinvar update, but it has been fixed. Please update to the latest version (2024.08.09) with `oc module update`.
